alternator/server.cc

```diff
@@ -319,10 +319,11 @@ future<> server::verify_signature(const request& req, const chunked_content& con
     });
 }
 
-static tracing::trace_state_ptr create_tracing_session() {
+static tracing::trace_state_ptr create_tracing_session(tracing::tracing& tracing_instance) {
```

Comment:
The fact you're now passing this tracing_instance parameter is just an optimization, to avoid calling get_local_tracing_instance() twice, right? This change wasn't really necessary here, right?
(I'm not against it, just trying to understand the motivation).

Reply:
It's for two things:
 - reducing the line length
 - removing the dependency on a global variable; it's still there one level up, but eventually we might want to remove that one as well